snap/squashfs/squashfs.go

```diff
@@ -587,3 +599,20 @@ func BuildDate(path string) time.Time {
 	t0, _ = time.Parse(time.ANSIC, matches[0][len(prefix):])
 	return t0
 }
+
+var truncateSnapToMinSize = func(snapPath string, minSize int64) error {
```

Comment:
I think it would be clearer if this were called `growSnapToMinSize` or something of the sort. To me, "truncate" has the connotation of shrinking, so it sounds like any snap above the minimum size is truncated to be equal to that minimum size. Since you're expanding any smaller snap to that minimum size and leaving anything larger untouched, this seems counterintuitive.